controllers/storagecluster/cephblockpools.go

```diff
@@ -15,19 +15,50 @@ import (
 
 type ocsCephBlockPools struct{}
 
+// ensures that peer cluster secret exists and adds it to CephBlockPool
```

Comment:
Is there a doc for the mirroring config? How many peers can a cephblockpool have?